web/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "serve": "vue-cli-service serve",
-    "prod": "vue-cli-service build"
+    "prod": "vue-cli-service build",
+    "prod-netlify": "vue-cli-service build --dest ./prod"
```

Comment:
Does netlify automatically pick up this one or do we have to set an option somewhere?

Reply:
Automatically picks it up apparently

Reply:
I already set the option on the Netlify panel